mxcube3/routes/Beamline.py

```diff
@@ -265,11 +265,27 @@ def prepare_beamline_for_sample():
     """
     Prepare the beamline for a new sample.
     """
-    
+
     try:
         if hasattr(mxcube.collect, 'prepare_for_new_sample'):
             mxcube.collect.prepare_for_new_sample()
     except Exception:
         logging.getLogger('HWR').error('Cannot prepare the Beamline for a new sample')
         return Response(status=200)
     return Response(status=200)
+
+@mxcube.route("/mxcube/api/v0.1/beamline/open_hutch", methods=['PUT'])
```

Comment:
We already have a hutchtrigger hardware object that can be configured through the beamline actions, what exactly do you want to achieve with this ?.

Reply:
that's rigth, but I have not way at this moment to link the hutch trigger with signals from the door opening PLC system. Moreover, our scientist request this button so that the user is more 'conscious' on this. I did it configurable so it only appears if you want to

Reply:
I see, this is exactly what the HutchTrigger hardware object is for and its possible to enable and disable this feature from the beamline actions menu. I would rather see that we use the beamline actions way unless you don't see any reasons for doing it differently or somehow altering beamline actions to suite your needs.

Reply:
I thought about the beamline actions thing but not much...
I can implement this as a bl action, but the bl actions are somehow hidden, and this is a action closely related to the queue operation / sample management and so on, that why we though on putting the button next to the queue.

Reply:
Hm, I maybe misunderstood what you want to do with your hutch trigger.

How is the hutch trigger closely related to the queue operation and sample management, its only triggered when you enter or exit the hutch ?, there should be neither of those going on when you are in the hutch ?

Reply:
With mxcube2 we have the hutchtrigger on/off as well, with the possibility to configured it as automatic or manual. The experience shows that it has never been used as manual, except for us doing tests of the software. For me hutchtrigger is more hoe to protect the equipment when somebody enters the hutch. It has lost its original idea of saving time when closing the hutch, as the actions are more related to mount/unmount sample. I do not think there should be a button next to the queue - this will only make the design crowded. Action, not related to a signal from PLC, are not really "hutchtrigger" related and are maybe better implemented in specific procedures, hidden from the users, as they will very likely forget to hit the button.